app/page.jsx

```diff
@@ -46,46 +47,49 @@ const Dashboard = () => {
 
   return (
     <div className="p-5">
-      <div className="mb-4">
-        <label htmlFor="view-by" className="mr-2">
-          View By:
-        </label>
-        <select
-          id="view-by"
-          value={viewBy}
-          onChange={handleViewByChange}
-          className="p-2 border rounded"
-        >
-          <option value="category">Category</option>
-          <option value="status">Status</option>
-        </select>
-      </div>
+      {tickets.length === 0 ? <NoTicketExistsCard /> :
+        <>
+          <div className="mb-4">
+            <label htmlFor="view-by" className="mr-2">
+              View By:
+            </label>
+            <select
+              id="view-by"
+              value={viewBy}
+              onChange={handleViewByChange}
+              className="p-2 border rounded"
+            >
+              <option value="category">Category</option>
+              <option value="status">Status</option>
+            </select>
+          </div>
 
-      <div>
-        {getUniqueGroups().map((group, groupIndex) => (
-          <div key={groupIndex} className="mb-4">
-            <h2>{group}</h2>
-            <div className="lg:grid grid-cols-2 xl:grid-cols-4 gap-4">
-              {tickets
-                .filter((ticket) =>
-                  viewBy === "category"
-                    ? ticket.category === group
-                    : ticket.status === group
-                )
-                .sort((a, b) => b.priority - a.priority)
-                .map((filteredTicket, ticketIndex) => (
-                  <TicketCard
-                    id={ticketIndex}
-                    key={ticketIndex}
-                    ticket={filteredTicket}
-                  />
-                ))}
-            </div>
+          <div>
+            {getUniqueGroups().map((group, groupIndex) => (
+              <div key={groupIndex} className="mb-4">
+                <h2>{group}</h2>
+                <div className="lg:grid grid-cols-2 xl:grid-cols-4 gap-4">
+                  {tickets
+                    .filter((ticket) =>
+                      viewBy === "category"
+                        ? ticket.category === group
+                        : ticket.status === group
+                    )
+                    .sort((a, b) => b.priority - a.priority)
+                    .map((filteredTicket, ticketIndex) => (
+                      <TicketCard
+                        id={ticketIndex}
+                        key={ticketIndex}
+                        ticket={filteredTicket}
+                      />
+                    ))}
+                </div>
+              </div>
+            ))}
           </div>
-        ))}
-      </div>
+        </>}
     </div>
-  );
```

Comment:
I presume this one was unintentional

Reply:
Sorry sir, i am not able to understand what. do you want to say.

Reply:
i think you removed the semicolon at line no 92 by mistake?

Reply:
Oh achha my bad 🥲. Resolved in latest commit.